tests/robustness/coverage/k8s_coverage_test.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+# Copyright 2025 The etcd Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs KIND (Kubernetes IN Docker) cluster to collect traces with
+# Jaeger for analysis in robustness/coverage tests.
+# It is based on instructions from tests/robustness/coverage/README.md
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# 1. Customize and set the environment variables
+export KUBERNETES_REPO="${KUBERNETES_REPO:-$(go env GOPATH)/src/k8s.io/kubernetes}"
+export ETCD_REPO="${ETCD_REPO:-$(go env GOPATH)/src/go.etcd.io/etcd}"
+export KUBECONFIG="${KUBERNETES_REPO}/kind-with-tracing-config"
+
+echo "Using KUBERNETES_REPO: ${KUBERNETES_REPO}"
+echo "Using ETCD_REPO: ${KUBERNETES_REPO}"
+echo "Using KUBECONFIG: ${KUBECONFIG}"
+
+echo "Applying patches to Kubernetes repo..."
+PATCHES_DIR="${ETCD_REPO}/tests/robustness/coverage/patches/kubernetes"
+pushd "${KUBERNETES_REPO}"
+git apply --reverse --check "${PATCHES_DIR}/"* || git apply --recount "${PATCHES_DIR}/"*
+echo "Building KIND node image..."
+kind build node-image
+popd
+
+echo "Creating Docker network..."
+docker network create kind-with-external-etcd \
+  --driver bridge \
+  --gateway "192.168.32.1" \
+  --subnet "192.168.32.0/24" || echo "Docker network already exists."
+rm_docker_network() {
+  docker network rm kind-with-external-etcd
+}
+trap "rm_docker_network" EXIT SIGINT
+
+echo "Starting Jaeger container..."
```

Comment:
Those command duplicate readme, I would prevent to avoid maintaining two copies of code. Maybe instead of script could you make stages be a Makefile targets?

Reply:
Unfortunately I need to keep some state between each makefile target stages and clean up when done. Replaced duplicated parts of README entry with `make` call

Reply:
I asked to remove the script, not the readme.